pkg/webhook/v1beta1/pod/inject_webhook.go

```diff
@@ -47,6 +48,13 @@ import (
 
 var log = logf.Log.WithName("injector-webhook")
 
+var (
+	errInvalidOwnerAPIVersion = errors.New("invalid owner API version")
+	errInvalidSuggestionName  = errors.New("invalid suggestion name")
+	errPodNotBelongToKatibJob = errors.New("pod does not belong to Katib Job")
+	errNestedObjectNotFound   = errors.New("nested object not found in the cluster")
```

Comment:
```suggestion
	errFailedToGetTrialTemplateJob  = errors.New("unable to get Job in the trialTemplate")
```

Reply:
This has been fixed in this [commit](https://github.com/kubeflow/katib/pull/2289/commits/f2b0ca4163afc29593144ee9d69612b7f0d00d6e).